data/mods/BombasticPerks/perkdata/frankenstein.json

```
@@ -0,0 +1,94 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "meat_hunk",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MEDICAL",
+    "skill_used": "firstaid",
+    "autolearn": false,
```

```suggestion
    "autolearn": false,
```
Autolearn by default is false

This recipe does need the SECRET flag like mutagens use so people can't cheat earning a perk by picking these skills at chargen.